scripts/cibuild.sh

```diff
@@ -24,11 +24,14 @@ fi
 # Check all built-in python source files for PEP 8 compliance, but explicitly
 # ignore:
 #  - E203: whitespace before ':' (because Black sometimes does this)
+#  - F401: module imported but unused (TODO: fix and re-enable this one)
+#  - F403: 'from module import *' used (TODO: fix and re-enable this one)
+#  - F405: name may be undefined or defined from star imports (TODO: fix and re-enable this one)
 #  - E501: line greater than 80 characters in length
 #  - W503: line break before binary operator
 #  - W504: line break after binary operator
-pycodestyle \
-    --ignore=E203,E501,W503,W504 \
+flake8 \
+    --ignore=E203,F401,F403,F405,E501,W503,W504 \
```

Comment:
IMO instead of using CLI flags let's move these to `.flake8` instead of having to maintain the `--ignore` flag and paths in two places (here and in the pre-commit hook).

Reply:
Good call!